test/gfx/input_tileset.flags

```diff
@@ -0,0 +1,2 @@
+-i input_tileset.in.2bpp
+-u
```

Comment:
I thought we only took the first line of .flags files? Is that behavior consistent between asm/fix/gfx/link?

Reply:
Not if they are passed as at-files. Some of the scripts that expand them manually maybe do?